drivers/sensor/pmw3360/pmw3360.c

```diff
@@ -628,7 +644,20 @@ static void irq_handler(struct device *gpiob, struct gpio_callback *cb,
 		.type = SENSOR_TRIG_DATA_READY,
 		.chan = SENSOR_CHAN_ALL,
 	};
+
 	handler(DEVICE_GET(pmw3360), &trig);
```

Comment:
I think you should protect it with `if (handler)`.

Reply:
Thanks! Fixed